my_dictionary.txt

```diff
@@ -0,0 +1,289 @@
+personal_ws-1.1 en 0
+acyclic
+afzal
+alertmanager
+allinone
+analytics
+anh
+anonymized
+anonymizer
+ansari
+ansh
+api
+apis
+apl
+ashmita
+async
+auth
+autoscale
+autoscaler
+backend
+backends
+bak
+balancer
+bansal
+bohara
+bootcamp
+cassandra
```

Comment:
what kind of standard dictionary are you using? I would've expected these words to be pre-defined somewhere.

Reply:
I was not using a standard dictionary, what I did was run the spellchecker once and I manually added all the words from the output which were misclassified as incorrect spellings but were technical terms or proper nouns. Should I also add more words from another source?